src/main/java/edu/ucsb/courses/controllers/StatisticsController.java

```diff
@@ -16,6 +16,8 @@
 import java.io.InputStreamReader;
```

Comment:
There are a LOT of unused imports in the Statistics Controller, this is not a result of your changes/PR. If you all feel up to it, you are welcome to go in and clean up the unused imports in StatisticsController.java

Reply:
We cleaned up all unused imports